providers.d/apt-mirror/provider_wrapper.sh

```diff
@@ -1,13 +1,37 @@
 bashelliteProviderWrapper() {
 
-  if [[ ! ${_r_dryrun} ]]; then
-    apt-mirror ${_r_metadata_tld}/repos.conf.d/${_n_repo_name}/provider.conf;
+  local basepath_parameter="$(grep -oP "(?<=(^set base_path[[:space:]]*))[[:alnum:]/]*" ${_r_metadata_tld}/repos.conf.d/${_n_repo_name}/provider.conf)"
+  if [[  "${basepath_parameter}" != "${_r_mirror_tld}/${_n_repo_name}" ]]; then
+    utilMsg WARN "$(utilTime)" "The \"base_path\" parameter (${basepath_parameter}) in provider.conf does not match mirror location (${_r_mirror_tld}/${_n_repo_name})..."
   fi
 
-  if [[ "${?}" != "0" ]]; then
-    utilMsg WARN "$(utilTime)" "apt-mirror either failed or completed with errors for repo (${_n_repo_name})."
+  local config_file="${_r_metadata_tld}/repos.conf.d/${_n_repo_name}/provider.conf"
+  local tmp_config_file="${HOME}/.bashellite/tmp_${_n_repo_name}_provider.conf"
+
+  cat /dev/null > ${tmp_config_file}
```

Comment:
No need to call an external binary. You can use the redirection operator by itself to truncate the file contents like so:

```> ${tmp_config_file}```

Reply:
K :)  That works for me.